files/ko/web/javascript/reference/global_objects/promise/allsettled/index.html

```diff
@@ -2,31 +2,80 @@
 title: Promise.allSettled()
 slug: Web/JavaScript/Reference/Global_Objects/Promise/allSettled
 translation_of: Web/JavaScript/Reference/Global_Objects/Promise/allSettled
+browser-compat: html.elements.allSettled
 ---
 <p>{{JSRef}}</p>
 
-<p><code><strong>Promise.allSettled()</strong></code> 메소드는 배열이나 별도의 나열 가능한 객체를 통해 나열된 Promise모음이 모두 이행하거나 거부했을 때에 대한 대응을 할 수 있는 Promise 객체를 반환한다.</p>
+<p><code><strong>Promise.allSettled()</strong></code> 메소드는 주어진 모든 프로미스를 이행하거나 거부한 후, 각 프로미스에 대한 결과를 나타내는 객체 배열을 반환합니다.</p>
+
+<p>일반적으로 서로의 성공 여부에 관련 없는 여러 비동기 작업을 수행해야 하거나, 항상 각 프로미스의 실행 결과를 알고 싶을 때 사용합니다.</p>
+
+<p>반면, {{jsxref("Promise.all()")}}이 반환한 프로미스는 서로 연관된 작업을 수행하거나, 하나라도 거부 당했을 때 즉시 거부하고 싶을 때 적합합니다.</p>
 
 <div>{{EmbedInteractiveExample("pages/js/promise-allsettled.html")}}</div>
 
-<h2 id="문법">문법</h2>
+<h2 id="syntax">문법</h2>
 
 <pre class="syntaxbox"><em>Promise</em>.allSettled(<em>iterable</em>);</pre>
 
-<h3 id="인자">인자</h3>
+<h3 id="parameters">인자</h3>
 
 <dl>
  <dt><code>iterable</code></dt>
- <dd><a href="/en-US/docs/Web/JavaScript/Guide/iterable">iterable</a> 객체로, 주로 배열({{jsxref("Array")}})을 사용하며, 이들의 요소들은 모두 <code>Promise</code> 객체.</dd>
+ <dd>멤버가 모두 <code>Promise</code>인, 배열({{jsxref("Array")}})과 같은 <a href="ko/docs/Web/JavaScript/Reference/Iteration_protocols">이터러블</a> 객체입니다.</dd>
 </dl>
 
-<h3 id="반환_값">반환 값</h3>
+<h3 id="return_value">반환 값</h3>
+
+<p>이행이나 거부 여부에 관계 없이 주어진 모든 프로미스가 완전히 실행된 후에 <strong>비동기</strong>적으로 이행될, <strong>대기 중</strong>인 프로미스를 반환합니다.</p>
```

Comment:
A pending Promise that will be asynchronously fulfilled once every promise in the specified collection of promises has completed, either by successfully being fulfilled or by being rejected. At that time, the returned promise's handler is passed as input an array containing the outcome of each promise in the original set of promises.

번역해주신 문장이 위의 문장과 다소 차이가 있는 것 같아 리뷰했습니다. 번역 시, MDN은 최대한 원문과 그대로 동일할 것을 권장합니다. 개개인의 문장 축소 및 요약은 권장하지 않습니다. 혹시, 해당 문장은 다시 고려해주실 수 있으신가요?

```suggestion
<p>지정된 Promise 컬렉션의 모든 Promise가 성공적으로 이행되거나 거부되어 완료되면, <strong>보류 중</strong>인 Promise는 <strong>비동기적</strong>으로 이행됩니다. 그 때, 반환된 Promise의 핸들러는 원래 프로미스 집합에 있는 각 프로미스의 결과를 포함하는 배열을 입력으로 전달합니다.</p>
```